test/lensinitUtils.js

```diff
@@ -40,7 +41,7 @@ describe('lensinitUtils test > ', () => {
 
   describe('copy packages test > ', function () {
 
-    this.timeout(7000);
+    this.timeout(10000);
```

Comment:
yikes! that's a long time! please add a comment explaining why it is necessary

Reply:
User won't need to do npm install, copy packages adds all the required npm modules

Reply:
This will make more sense when LDK is installed as a node package I haven't changed the install part in README since it hasn't been published